vernac/vernacextend.ml

```diff
@@ -238,7 +238,7 @@ let warn_deprecated_command =
 
 (* Interpretation of a vernac command *)
 
-let type_vernac opn converted_args ?loc ~atts () =
```

Comment:
This kind of removal is causing the fatal warning 16s

Reply:
So why does it build fine locally?

Reply:
(I don't want to keep this removal, just trying to align my local build setup with CI)

Reply:
It builds fine locally because your version of OCaml is old and therefore the warning is not fatal.

Reply:
The "fatality" of the warning is defined by the OCaml version? And btw, I'm using the OCaml version that comes with the Coq Nix dev setup.

Reply:
old ocaml does not detect this issue (ie there is no warning), it's not about fatality